var/spack/repos/builtin/packages/py-genshi/package.py

```diff
@@ -16,7 +16,7 @@ class PyGenshi(PythonPackage):
     depends_on("py-setuptools", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools", type=("build", "run"))
```
See if this fixes the tests

Reply:
Yes it does, thanks!